GSoC/2016/README.md

```diff
@@ -0,0 +1,29 @@
+# Note for 2016 GSoC application
+
+It's my first time in the GSoC application. Due to my lack of the process, I missed the deadline of updating my enrollment letter, which finally caused the eligibility.
+
+This proposal was not selected as one of the GSoC project as the enrollment letter was not in the correct form. I didn't know this massage until I checked the GSoC website manually before the last day which was too late according to the rules. So unfortunately, this project will not be eligible.
+
+However, the project will go on, if anyone have interests in this idea, please check [QuCmp.jl](https://github.com/JuliaQuantum/QuCmp) in JuliaQuantum's repo.
+
+To prevent the same accident from happening again and to help people in region without Google services to get involved with all those amazing ideas, I'll leave some advise here for other students from where google is not available :
```

Comment:
To prevent the same tragedy happening on future applicants, especially to help applicants in regions without Google services, I am leaving some advice below.